src/js/contentUtils.ts

```diff
@@ -234,8 +280,8 @@ export function storeFoundJS(scriptNodeMaybe: HTMLScriptElement): void {
           STATES.INVALID,
           `No data-btmanifest attribute found on script ${scriptNodeMaybe.src}`,
         );
+        invalidateAndThrow('Script is missing `data-btmanifest` prop');
```

Comment:
Remove the state update above since we have this now? But let's keep the `script.src` to help with debugging.